Extensions/DialogueTree/dialoguetools.js

```diff
@@ -420,7 +428,7 @@ gdjs.dialogueTree.startFrom = function(startDialogueNode) {
   if (!this.hasDialogueBranch(startDialogueNode)) return;
   this.optionsCount = 0;
   this.options = [];
-  this.dialogueBranchTitle = '';
+  this.dialogueBranchTitle = null;
```

Comment:
What does this changes do?

Reply:
doesnt really change anything. I didnt like the way I was checking this when fixing these bugs. I can undo this chage

Reply:
I now changed this logic completely to properly initialize these values instead of starting them out as nulls